medical_procedure_external/models/medical_procedure_external_request.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2022 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class MedicalProcedureExternalRequest(models.Model):
+    _name = "medical.procedure.external.request"
+    _inherit = ["medical.event", "digest.base"]
+    _description = "Procedure external"
+    _rec_name = "internal_identifier"
+
+    @api.model
+    def _get_states(self):
```

Comment:
What do you think about calling the states "draft", "final" (or another name), and "canceled"?

Reply:
I don't know, the current ones are also fine, as you prefer!